cli/tests/test_git_push.rs

```diff
@@ -574,8 +572,7 @@ fn test_git_push_deleted() {
     insta::assert_snapshot!(stdout, @r###"
     Branch changes to push to origin:
       Delete branch branch1 from 45a3aa29e907
-    Abandoned the following commits:
-      lzmmnrxq 45a3aa29 branch1@origin | (empty) description 1
+    Abandoned 1 commits.
```

Comment:
You should probably special-case 1 and print "Abandoned 1 commit."

Reply:
I think we generally avoid special case for singular form. That probably comes from Mercurial.

Reply:
Hmm... I've always done the opposite. I thought `jj`'s way is to try to be as close to proper English as possible. For example, we have `make_branch_term` in

https://github.com/martinvonz/jj/blob/dbbed2d2a5f09b7adeab3e1caf85b274d50d9082/cli/src/commands/mod.rs#L3636

For the purposes of this PR, I don't think this is particularly important, though.

Reply:
FWIW, I have preferred keeping code simple over using the right singular/plural form. For example: 
https://github.com/martinvonz/jj/blob/dbbed2d2a5f09b7adeab3e1caf85b274d50d9082/cli/src/cli_util.rs#L1462

I don't mind if people want to make it prettier. I just prefer the simpler code myself (and simpler i18n if we do that). Maybe I've gotten used to that from Mercurial as Yuya said.

Reply:
I also prefer simpler code, and I don't think using the proper singular/plural form is important for programmer tools.